eureka2-client/src/main/java/com/netflix/eureka2/client/resolver/HostResolverStep.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.eureka2.client.resolver;
+
+/**
+ * @author David Liu
+ */
+public interface HostResolverStep {
```

Comment:
"Server" in our terminology implies hostname + port and I wanted to avoid that implication, hence "HostResolver".